### DISCUSSION TITLE
Shelly- oder Tasmota- Plugs mit in Netzbezug einbrechen?

### DISCUSSION THREAD

**Discussion by rs3hc** [hoylabs/OpenDTU-OnBattery#695] [General]
Hallo Leute.

Besteht die Möglichkeit den Verbrauch eines Shelly- oder Tasmota- Plugs mit in den Netzbezug einzubrechen?

Aktuell habe ich im Wohnungsverteilen einen Shelly 3em der auch ordnungsgemäß funktioniert und in OpdenDTU erkannt wird.

Allerdings habe ich im Keller noch Verbraucher, die nicht mit gemessen werden.

Kann man die irgendwie mit aufaddieren?

**Comment by spcqike** [comment]
Du kannst beim http(s)+json powermeter einzelne Phasen definieren und diesen unterschiedliche URLs und auch json Pfade geben. Damit kannst du probieren, ob beide geräte abgefragt werden können und der Verbrauch summiert wird.
